tests/Maintenance/MaintenanceTest.cpp

```diff
@@ -98,8 +98,6 @@ size_t localId = 1016002;
 std::string S("s");
```

Comment:
Sorry this file suffered from reformatting, I have added comments an actually modified code